client/src/Components/ReviewsContainer/List/ReviewList.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react"
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
+import Button from "react-bootstrap/Button"
+import ReviewTile from "./ReviewTile.jsx"
+import useReviews from "../../../contexts/ReviewsContext.jsx"
+
+const ReviewList = (props) => {
+  const { reviews, fetchReviews } = useReviews()
+  const [page, setPage] = useState(1)
+  const [count, setCount] = useState(2)
+  const [sort, setSort] = useState("relevant") // based on sort
+  const [product_id, setProduct_id] = useState(42367)
+  const [tiles, setTiles] = useState([])
+  const [next, setNext] = useState([])
+
+  // create first set of tiles
+  useEffect(() => {
+    fetchReviews(page, count, sort, product_id, (data) => {
+      setTiles(
+        data.results.map((review) => (
```

Comment:
will the reviews array from useReview() be updated when you fetch here?

If the reviews array in stored/handled in ReviewsContext, and then map the reviews list inside on <Container> below